apps/docs/pages/guides/auth/auth-mfa.mdx

```diff
@@ -241,6 +245,82 @@ export function EnrollMFA({
 }
 ```
 
+### Add unenrollment flow
+
+An unenrollment flow provides a UI for users to manage and unenroll factors linked to their accounts.
+Most applications do so via a factor management page where users can view and unlink selected factors.
+
+To unenroll a factor, a user simply needs to call `supabase.auth.mfa.unenroll()` with the id of the factor they wish to unenroll.
+For example, to unenroll the nil UUID we would call: `supabase.auth.mfa.unenroll({factorId: "00000000-0000-0000-0000-000000000000"})`.
+
+
+#### Example: React
+
+Below is an example that creates a new `UnenrollMFA` component that illustrates
+the important pieces of the MFA enrollment flow. Note that a user can only unenroll factors they own and they can only unenroll factors after
+first going through the MFA process and attaining the `aal2` level. Here are some points of note:
+
+- When the component appears on screen, the `supabase.auth.mfa.listFactors()` API is
+  called once to fetch all existing factors together with their details.
+- The existing factors for a user are displayed in a table.
+- Once the user has selected a factor to unenroll they can type in the factorId and hit the `Unenroll` button
+  which creates a confirmation modal.
+
+<Admonition type="note">
+A user can only unenroll a factor after completing the enrollment flow and obtaining
+an `aal2` JWT claim.
+</Admonition>
```

Comment:
This is pretty similar to the statement in the paragraph above—should get rid of one of them.